ipapython/install/cli.py

```diff
@@ -179,6 +179,7 @@ def add_options(cls, parser, positional=False):
             elif issubclass(knob_scalar_type, enum.Enum):
                 kwargs['type'] = 'choice'
                 kwargs['choices'] = [i.value for i in knob_scalar_type]
+                kwargs['metavar'] = "[%s]" % ("|".join(kwargs['choices']))
```

Comment:
- Please use the same format as `argparse`: `{a,b,c}`.
- [`str.format()`](https://docs.python.org/3/library/stdtypes.html#str.format) is the preferred way of formatting strings.